wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -45,7 +88,7 @@ function _pantheon_upstream_update_notice() {
 	$update_type = 'new WordPress version';
 	$update_help = 'If you need help, open a support chat on Pantheon.';
 	$data = _pantheon_curl_cached( 'https://api.live.getpantheon.com/sites/self/code-upstream-updates' );
-	if ( ! empty( $data['remote_url'] ) && false === stripos( $data['remote_url'], '/pantheon-systems/' ) ) {
+	if ( ! _pantheon_is_default_upstream() ) {
 		$update_type = 'Pantheon Custom Upstream update';
```

Comment:
If Pantheon *isn't* the default upstream, we change the message to 'Pantheon Custom Upstream update'? Did I read the code wrong? Seems backwards by inspection. Is the message correct when running?

Reply:
if using the default Pantheon WordPress upstream, the message is `"new WordPress Version"`, "Pantheon Custom Upstream update" is only if the site is using a different upstream (i.e. one maintained by an org). May be more clear to have "Pantheon Upstream" vs "Pantheon Custom Upstream", but I think I prefer as written (`new WordPress version`).

Reply:
Could we maybe rename / refocus `_pantheon_is_default_upstream()` to return the org/project for the upstream, or `false` if the upstream is not a custom upstream (that is, if it is pantheon-systems/WordPress)? Then we could put the org/project of the custom upstream in the message. That would be a lot clearer, and very helpful. `_pantheon_is_default_upstream()` is already doing a web request that returns the remote_url.

Reply:
Updated function & now include org/repo in the message if using a custom upstream.

<img width="904" alt="Screen Shot 2020-04-16 at 1 41 51 PM" src="https://user-images.githubusercontent.com/8714062/79489268-07038480-7fe9-11ea-9e49-c43b8e782a92.png">